### PR TITLE
TC_12.002.04 | Footer > REST API link > Visibility and clickability

### PR DESCRIPTION
**Pull request by MariannaLiss**
https://trello.com/c/8aEwN5er/612-tc1200204-footer-rest-api-link-visibility-and-clickability

### CODE REVIEW

File: src/test/java/school/redrover/RestApiLink3Test.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+public class RestApiLink3Test extends BaseTest {
+    @Test
+    public void testVerifyRestApiLink() {
+        String apiLink = "REST API";
```
Comment (EkaterinaSokolova-V) [inline src/test/java/school/redrover/RestApiLink3Test.java:11]: Please add final for apiLink variable.

**Comment by EkaterinaSokolova-V** [comment]
Step 2 looks like AC, please remove it.
Don't forget add PR link to your TC card.

**Comment by MariannaLiss** [comment]
> Step 2 looks like AC, please remove it. Don't forget add PR link to your TC card.

Have done

File: src/test/java/school/redrover/RestApiLink3Test.java (hunk 1 above)
Comment (MariannaLiss) [reply src/test/java/school/redrover/RestApiLink3Test.java]: Have done

**Review by EkaterinaSokolova-V — APPROVED** [review]

**Review by Vlad1kek — APPROVED** [review]

**Pull request merged**